enterprise-suite/templates/prometheus.yaml

```diff
@@ -1,4 +1,21 @@
 {{- $shouldCreatePVCs := and (or .Values.usePersistentVolumes (not .Values.useEmptyDirVolumes)) (.Values.createPersistentVolumes) }}
+
+# The following crazyness simply adds the port suffix to each service in a list and
+# joins them into a string with commas. Argument is a tuple (list, port). Eg.:
+#
+# test:
+#   - service-00
+#   - service-01
+# {{ tuple .Values.test "9000" | include "serviceList" }}
+#
+# Evaluates to:
+# service-00:9000,service-01:9000
+{{- define "serviceList" -}}
+{{- $suffix := index . 1 -}}
+{{- $local := dict "first" true -}}
+{{- range $k, $v := index . 0 -}}{{- if not $local.first -}},{{- end -}}{{- $v -}}:{{- $suffix -}}{{- $_ := set $local "first" false -}}{{- end -}}
+{{- end -}}
+
```

Comment:
Can you put this into `_helpers.tpl`, that's the helm convention.

Reply:
Done